src/app/components/rangeselector/rangeselector.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { DataService } from '../../services/data.service';
+import { DatePipe } from '@angular/common';
```

Comment:
Can we use the built-in LDML pattern support here instead of DatePipe?

Reply:
It is not published yet

Reply:
OK